### DISCUSSION TITLE
activation problem

### DISCUSSION THREAD

**Discussion by DjKost7** [rshea0/enjinn#61] [Help] [answered]
i bought and tried to activate licence key. but theres an error message.

**Comment by shayded-exe** [comment]
Sorry, just saw this. Please email me your license key at shayded@shayded.com

**Comment by shayded-exe** [comment, marked as answer]
This has been fixed in [v2.2.0](https://github.com/rshea0/tuneup-prime/releases/tag/v2.2.0)
